images/metering-ansible-operator/roles/meteringconfig/tasks/reconcile.yml

```diff
@@ -14,8 +14,16 @@
   vars:
     hiveStorageType: "{{ meteringconfig_spec_overrides | json_query('storage.hive.type') }}"
 
+- name: Validate fields when OCP features are disabled
+  assert:
+    that:
+      - meteringconfig_spec | json_query('reporting-operator.spec.config.prometheus.url') != ""
```

Comment:
@chancez I put this check in place to ensure that the URL is set, and wasn't sure if we wanted to use the default name (`url: https://prometheus-k8s.openshift-monitoring.svc:9091/`). I can reset the name of this field in the override variables to enforce this.

Reply:
See my other comment. I think validating it makes sense. I also think _not_ having a defalt for non-OCP is probably best.

Perhaps we should unset the prometheus.url values.yaml default, and use an override to set the default for OCP, and then it's unset for non-OCP (which will fail validation if they don't set the prometheus URL).